prover/proof_submitter/util.go

```diff
@@ -72,6 +72,7 @@ func sendTxWithBackoff(
 	retryInterval time.Duration,
 	maxRetry *uint64,
 	waitReceiptTimeout time.Duration,
+	proveBlocktxGasLimit *uint64,
```

Comment:
Seems this new parameter is not used?

Reply:
ah yeah it needs to be removed from there, i had added it before i saw we had `sendTx` func :D